pkg/runner/exec.go

```diff
@@ -35,8 +38,8 @@ import (
 
 var lock = &sync.Mutex{}
 
-func runBenchmark(cfg config.Config, clusterMetadata tools.ClusterMetadata) ([]tools.Result, error) {
-	var aggAvgRps, aggAvgLatency, aggP99Latency float64
+func runBenchmark(cfg config.Config, clusterMetadata tools.ClusterMetadata, p *prometheus.Prometheus) ([]tools.Result, error) {
+	var aggAvgRps, aggAvgLatency, aggP95Latency float64
```

Comment:
Why are switching to P95?

Reply:
> Why are switching to P95?

We've agreed with the network-edge team to use this metrics rather than P99, as its more stable